core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -354,19 +354,27 @@ object KafkaConfig {
   /************* Authorizer Configuration ***********/
   val AuthorizerClassNameDoc = "The authorizer class that should be used for authorization"
   /** ********* Socket Server Configuration ***********/
-  val PortDoc = "the port to listen and accept connections on"
-  val HostNameDoc = "hostname of broker. If this is set, it will only bind to this address. If this is not set, it will bind to all interfaces"
+  val PortDoc = "DEPRECATED: only used when \"listeners\" is not set. " +
+  "Use \"listeners\" instead. \n" + 
```

Comment:
I suggest using backquotes (eg `) instead of double quotes as it nicely avoids the escaping problem.